### ISSUE TITLE
IE Freezes during Fetch

### ISSUE THREAD

**Issue by gsomoza** [Strategery-Inc/Magento-InfiniteScroll#129]
I am currently using the Strategery Infinite Scroll extension for Magento CE 1.9.2.1. So far I have enjoyed the plugin, but I’m running into an issue and was not able to find anything online about it. When browsing the site in any IE browser, even the most current, the infinite scroll freezes the browser for up to a minute or two before finally appending the products. 

I have narrowed it down to the fact that somehow the way jQuery.after() is being called, IE loads everything in synchronously instead of asynchronously. This is causing a huge delay in the append, and causing the browser to freeze up during the process. We have tried debugging this issue ourselves but as of now have not been able to find a solution.

**Comment by gabrielsomoza** [comment]
@damian-pastorini could you look into this? High priority.

**Comment by damian-pastorini** [comment]
I can't reproduce the issue, I installed a clean Magento 1.9.2.1 with the sample data, then installed the InfiniteScroll through the Connect Manager and tested in IE 11, the screen didn't freeze up at any point.
Also, as side note, the only place where the jQuery.after() method is implemented is inside the ias library.
Do you have an URL where I can see the issue? 
Maybe you have a custom theme or something else is causing the error? Maybe something like another jQuery version active from the one that's included in the extension?

**Comment by rain2o** [comment]
Yes it is using a custom theme. You can see the issue here - http://patricianashdesigns.com/women/handbags. 
We are using jQuery 2.1.4 instead of the version with the extension.

**Comment by damian-pastorini** [comment]
Hi @rain2o , could you try with the default extension? Becuase this seems to be a jQuery issue with IE.

**Comment by rain2o** [comment]
I have spent some time diving into this and found that most of the issues did ultimately come back to my theme. However, there is a small change that I had to make to the `init.phtml` file to allow me to accomplish my goals more optimally. I have submitted pull request #139 with the changes. I just triggered publicly accessible events so that my JS could bind to those events for additional functionality. 

Thanks for looking into this for me!
